2021-12-28 14:50:44,398 INFO: Initialized curve data.
2021-12-28 14:50:44,401 INFO: Setting up initial twist data...
2021-12-28 14:50:46,085 INFO: Twist setup complete.
2021-12-28 14:50:46,085 INFO: Searching for rational points on each twist...
2021-12-28 14:50:48,348 INFO: Finished searching for rational points on each twist.
2021-12-28 14:50:48,348 INFO: Testing local solvability of twists...
2021-12-28 14:50:54,558 INFO: Finished local solvability testing.
2021-12-28 14:50:54,560 INFO: Computing elliptic curve a-invariants...
2021-12-28 14:50:59,298 INFO: Finished computation of a-invariants.
2021-12-28 14:50:59,300 INFO: Computing MW group (delta = ['-4', '4', '15', '7', '3', '2'], g = [-1, 1, 1])...
2021-12-28 14:51:01,295 INFO: Finished MW computation (delta = ['-4', '4', '15', '7', '3', '2'], g = [-1, 1, 1])
2021-12-28 14:51:01,297 INFO: Computing MW group (delta = ['-4', '4', '15', '7', '3', '2'], g = [-2, 5, -1, 1])...
2021-12-28 14:51:26,323 INFO: Finished MW computation (delta = ['-4', '4', '15', '7', '3', '2'], g = [-2, 5, -1, 1])
2021-12-28 14:51:26,325 INFO: Computing MW group (delta = ['0', '-8', '4', '3', '0', '1'], g = [-1, 1, 1])...
2021-12-28 14:51:27,391 INFO: Finished MW computation (delta = ['0', '-8', '4', '3', '0', '1'], g = [-1, 1, 1])
2021-12-28 14:51:27,393 INFO: Computing MW group (delta = ['0', '-8', '4', '3', '0', '1'], g = [-2, 5, -1, 1])...
2021-12-28 14:51:31,655 INFO: Finished MW computation (delta = ['0', '-8', '4', '3', '0', '1'], g = [-2, 5, -1, 1])
2021-12-28 14:51:31,658 INFO: Skipped MW computation because no base point (delta = ['3', '-7', '-2', '-1', '-1'])
2021-12-28 14:51:31,658 INFO: Computing MW group (delta = ['1', '-1'], g = [-1, 1, 1])...
2021-12-28 14:51:33,418 INFO: Finished MW computation (delta = ['1', '-1'], g = [-1, 1, 1])
2021-12-28 14:51:33,421 INFO: Computing MW group (delta = ['1', '-1'], g = [-2, 5, -1, 1])...
2021-12-28 14:51:51,535 INFO: Finished MW computation (delta = ['1', '-1'], g = [-2, 5, -1, 1])
2021-12-28 14:51:51,539 INFO: Computing MW group (delta = ['3', '-10', '5', '1', '0', '1'], g = [-1, 1, 1])...
2021-12-28 14:51:53,132 INFO: Finished MW computation (delta = ['3', '-10', '5', '1', '0', '1'], g = [-1, 1, 1])
2021-12-28 14:51:53,134 INFO: Computing MW group (delta = ['3', '-10', '5', '1', '0', '1'], g = [-2, 5, -1, 1])...
2021-12-28 14:52:00,507 INFO: Finished MW computation (delta = ['3', '-10', '5', '1', '0', '1'], g = [-2, 5, -1, 1])
2021-12-28 14:52:00,511 INFO: Computing MW group (delta = ['1'], g = [-1, 1, 1])...
2021-12-28 14:52:03,477 INFO: Finished MW computation (delta = ['1'], g = [-1, 1, 1])
2021-12-28 14:52:03,480 INFO: Computing MW group (delta = ['1'], g = [-2, 5, -1, 1])...
2021-12-28 14:52:29,930 INFO: Finished MW computation (delta = ['1'], g = [-2, 5, -1, 1])
2021-12-28 14:52:29,933 INFO: Skipped MW computation because no base point (delta = ['4', '-16', '13', '12', '2', '3'])
2021-12-28 14:52:29,933 INFO: Computing MW group (delta = ['4', '-20', '13', '9', '0', '3'], g = [-1, 1, 1])...
2021-12-28 14:52:31,730 INFO: Finished MW computation (delta = ['4', '-20', '13', '9', '0', '3'], g = [-1, 1, 1])
2021-12-28 14:52:31,732 INFO: Computing MW group (delta = ['4', '-20', '13', '9', '0', '3'], g = [-2, 5, -1, 1])...
2021-12-28 14:52:39,914 INFO: Finished MW computation (delta = ['4', '-20', '13', '9', '0', '3'], g = [-2, 5, -1, 1])
2021-12-28 14:52:39,919 INFO: Reducing MW generators...
2021-12-28 14:52:40,254 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 537, in <module>
    reduced_gens = reduce_MW_gens(gens_mag)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 316, in reduce_MW_gens
    P += T[i][j] * indep_gens[j]
  File "sage/matrix/matrix0.pyx", line 1035, in sage.matrix.matrix0.Matrix.__getitem__ (build/cythonized/sage/matrix/matrix0.c:8317)
    r = self.row(row)
  File "sage/matrix/matrix1.pyx", line 1255, in sage.matrix.matrix1.Matrix.row (build/cythonized/sage/matrix/matrix1.c:14161)
    V = sage.modules.free_module.FreeModule(self._base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 474, in FreeModule
    return FreeModuleFactory_with_standard_basis(base_ring, rank, sparse,
  File "sage/structure/factory.pyx", line 369, in sage.structure.factory.UniqueFactory.__call__ (build/cythonized/sage/structure/factory.c:2258)
    return self.get_object(version, key, kwds)
  File "sage/structure/factory.pyx", line 412, in sage.structure.factory.UniqueFactory.get_object (build/cythonized/sage/structure/factory.c:2652)
    obj = self.create_object(version, key, **extra_args)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 254, in create_object
    return FreeModule_ambient_pid(base_ring, rank, sparse=sparse)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 5522, in __init__
    FreeModule_ambient_domain.__init__(self, base_ring=base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 5331, in __init__
    FreeModule_ambient.__init__(self, base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 4694, in __init__
    FreeModule_generic.__init__(self, base_ring, rank=rank,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 828, in __init__
    self.Element = element_class(coordinate_ring, sparse)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 7349, in element_class
    import sage.modules.vector_real_double_dense
ImportError: /share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/vector_real_double_dense.cpython-39-x86_64-linux-gnu.so: failed to map segment from shared object: Cannot allocate memory
